ehitchhiking-ui/src/app/shared/components/blacklist/blacklist.component.sass

```diff
@@ -0,0 +1,36 @@
+@import "../../../../styles/colour-constants"
+@import "../../../../styles/fonts-constants"
+
+.truncated
+  white-space: nowrap
+  overflow: hidden
+  text-overflow: ellipsis
+.blacklist-title
+  position: relative
+  left: 40%
```

Comment:
Why?

Reply:
so that the back button is in the middle

Reply:
fixed